src/promptflow/promptflow/storage/_sqlite_client.py

```diff
@@ -1,168 +0,0 @@
-# ---------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# ---------------------------------------------------------
-
-import dataclasses
-import os
-from dataclasses import asdict, fields
-from datetime import datetime
-from sqlite3 import OperationalError
-from typing import List
-
-import dataset
-from dataset.types import Types
-from sqlalchemy.exc import IntegrityError
-
-from promptflow._utils.retry_utils import retry
-from promptflow.storage._errors import DuplicatedPrimaryKeyException, NotFoundException
-
-PRIMARY_KEY = "PrimaryKey"
-INDEX = "INDEX"
```

Comment:
Can we keep _sqlite_client.py and _errors.py?

Otherwise runtime will need a lot of change, since LocalRunStorage depends on _sqlite_client.py and a lot of tests (localmode) and runtime logic depend on LocalRunStorage.